nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -200,6 +205,34 @@
               + FL_ACKNOWLEDGE_RESTRICTIVE
               + " is also set");
     }
+    fixSerializationActivationFlag = flags.getBoolean(FL_FIX_SERIALIZATION).orElse(false);
+    Optional<String> autoFixOutPutDir = flags.get(FL_FIX_SERIALIZATION_OUTPUT_DIR);
+    if (fixSerializationActivationFlag && !autoFixOutPutDir.isPresent()) {
+      throw new IllegalStateException(
+          "DO NOT report an issue to Error Prone for this crash!  NullAway Fix Serialization configuration is "
+              + "incorrect.  "
+              + "Must specify AutoFixer Output Directory, using the "
+              + "-XepOpt:"
+              + FL_FIX_SERIALIZATION_OUTPUT_DIR
+              + " flag.  If you feel you have gotten this message in error report an issue"
+              + " at https://github.com/uber/NullAway/issues.");
+    }
+    /*
+     * if fixSerializationActivationFlag is false, the default constructor is invoked for
+     * creating FixSerializationConfig which all features are deactivated, this is for keeping
+     * it @Nonnull.
```

Comment:
```suggestion
     * creating FixSerializationConfig which all features are deactivated.  This lets the
     * field to be @Nonnull, allowing us to avoid null checks in various places.
```

Reply:
732f7ad24090d81fb686dfd2b90e84e7d183a94b